rpc/commands/commands.proto

```diff
@@ -218,10 +221,10 @@ message UpgradeReq {
 }
 
 message UpgradeResp {
-	// Progress of the downloads of the platforms and libraries files.
-	DownloadProgress progress = 1;
-	// Description of the current stage of the upgrade.
-	TaskProgress task_progress = 2;
+  // Progress of the downloads of the platforms and libraries files.
+  DownloadProgress progress = 2;
+  // Description of the current stage of the upgrade.
+  TaskProgress task_progress = 3;
```

Comment:
Why this change?

Reply:
No idea, I didn't manually change it, it's probably some auto formatting that I missed.

Reply:
It's not about the indentation; this is a change from `1` to `2`, and from `2` to `3`. Just asking though...

Reply:
Yes, I noticed it. Still not sure how it did happen. :shrug: